pyrgg/__main__.py

```diff
@@ -26,15 +26,17 @@
     15: gexf_maker}
 
 
-def run():
+def gen_graph(input_dict, file_name):
     """
-    Run proper converter.
+    Generate a single graph.
 
+    :param input_dict: input data
+    :type input_dict: dict
+    :param file_name: file name
+    :type file_name: str
     :return: None
```

Comment:
It would be nice if we return the generated graph as an object and pass it to another function to print it, it can help us for future moves about changing the structure from so used program to library form.

I'll open an issue about it if you have the same altitude toward it.

Reply:
Thanks for your review 💯 
No new Issue needed, we will solve it in a future release.